examples/nftbillboard-userdata/packages/hardhat/contracts/RedirectAll.sol

```diff
@@ -28,8 +28,6 @@ import {
 
 contract RedirectAll is SuperAppBase {
```

Comment:
CFA Hash identifier: Use the same pattern when possible.
Remove unused imports

Reply:
@ngmachado this has been fixed and is ready for your review